src/CMakeLists.txt

```diff
@@ -15,15 +26,20 @@ target_compile_definitions(journald
         -DG_LOG_DOMAIN="eventd-journald")
 target_link_libraries(journald
     PRIVATE
-        ${EventdEvent_LIBRARIES}
+        ${Eventd_LIBRARIES}
         ${EventdPlugin_LIBRARIES}
+        ${EVENTDHELPERS_LIBRARY}
         ${SystemdJournal_LIBRARIES})
 set_target_properties(journald
     PROPERTIES
         PREFIX    ""
         NO_SONAME 1)
 
+pkg_config_variable(libeventd-plugin pluginsdir EVENTD_PLUGINS_DIR_INTERNAL)
+set(EVENTD_PLUGINS_DIR ${EVENTD_PLUGINS_DIR_INTERNAL}
```

Comment:
If you build a plugin against a specific eventd, it must go to the `pluginsdir` of this specific eventd. Otherwise, API and ABI may not match, crashes will occur and so on. If you want to put a plugin in a different prefix, then there must be an eventd there too, and you just have to point pkg-config to that specific eventd. Simple, isn’t it? :-)